Source/WebCore/platform/mediastream/ios/AVAudioSessionCaptureDeviceManager.mm

```diff
@@ -40,6 +40,9 @@
 
 #import <pal/cocoa/AVFoundationSoftLink.h>
 
+SOFT_LINK_PRIVATE_FRAMEWORK_OPTIONAL(AudioSession)
+SOFT_LINK_CONSTANT(AudioSession, AVAudioSessionPortBuiltInMic, NSString *)
```

Comment:
It looks like the changes in this file are for rdar://96253856?

Reply:
Right, will revert it.